### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by jessmathews**
rename files to convention.
cleaned up README.md

### CODE REVIEW

**Comment by jessmathews** [comment]
pls merge uwu

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-Objectives
+## Objectives
 
-> extract coordinates of pixels from a binary image
-> map out the points to real-life-size paper dimensions (in cm)
-> insert the points into a readable file format for the arduino
-> create a arduino program to control the motors to move the stylus
-> create a 3D CAD file of the 2D Plotter machine
+- [ ]  extract coordinates of pixels from a binary image
```
Comment (jessmathews) [inline README.md:6]: it makes the objectives behave like checkboxes. Just put an x inside the space when the task is done.

Comment (jyttr) [reply README.md:6]: Thanks brother

**Comment by jyttr** [comment]
thanks for contributing.

**Pull request closed without merging**